### PR TITLE
#165273563 Setup Heroku Pipelines 

### PR DESCRIPTION
**Pull request by llwasampijja**
#### What does this PR do?
Setup, Heroku
#### Description of Task to be completed?
The task included setting up Heroku, deploying to heroku and setting up the necessary configurations!
#### How should this be manually tested?
This can be tested by postman using the following endpoints for signing up and logging in:
* https://ah-backend-dojo-dev.herokuapp.com/api/users/
* https://ah-backend-dojo-dev.herokuapp.com/api/users/login/
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#165273563](https://www.pivotaltracker.com/story/show/165273563)

@AtamaZack @bisonlou @kalsmic @nadralia @JamesMudidi

### CODE REVIEW

**Comment by phillipseryazi** [comment]
@llwasampijja fix these merge conflicts as well.

**Comment by kalsmic** [comment]
Well done @llwasampijja, it seems the migrations did not run on Heroku.
Screenshot 2019-04-24 at 16 55 28
Can you try running migrations via the Heroku command line

**Review by ZackAtama — APPROVED** [review]
Awesome work @llwasampijja !!

**Review by bisonlou — APPROVED** [review]
Great work @llwasampijja. I have tested the register and login endpoints and they work.

File: .env_sample [hunk 1]
```diff
@@ -1,5 +1,5 @@
-export DB_NAME=''
-export DB_USER=''
-export DB_PASS=''
-export DB_HOST=''
-export DB_PORT=''
\ No newline at end of file
+export DB_NAME='dojo'
```
Comment (nadralia) [inline .env_sample:1]: @llwasampijja  
Remove your sample data , export DB_NAME='' not export DB_NAME='dojo'

**Review by nadralia — COMMENTED** [review]
Nice Work, The heroku integration worked well

**Comment by phillipseryazi** [comment]
@llwasampijja please make sure your pr is named correctly

**Review by nadralia — APPROVED** [review]

**Pull request merged**